lib-app-bitcoin/Makefile

```diff
@@ -0,0 +1,143 @@
+# ****************************************************************************
+#    Ledger App Bitcoin
+#    (c) 2023 Ledger SAS.
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+# ****************************************************************************
+
+ifeq ($(BOLOS_SDK),)
+$(error Environment variable BOLOS_SDK is not set)
+endif
+include $(BOLOS_SDK)/Makefile.defines
+
+########################################
+#        Mandatory configuration       #
+########################################
+# Name is defined later
+
+ifeq ($(APPVERSION_M),)
+$(error APPVERSION_M must be defined)
+endif
+ifeq ($(APPVERSION_N),)
+$(error APPVERSION_N must be defined)
+endif
+ifeq ($(APPVERSION_P),)
+$(error APPVERSION_P must be defined)
+endif
+
+APPVERSION = "$(APPVERSION_M).$(APPVERSION_N).$(APPVERSION_P)"
+
+APP_SOURCE_PATH += lib-app-bitcoin/
+
+ICON_NANOS = icons/nanos_app_$(COIN).gif
+ICON_NANOX = icons/nanox_app_$(COIN).gif
+ICON_NANOSP = icons/nanox_app_$(COIN).gif
+ICON_STAX = icons/stax_app_$(COIN).gif
+ICON_FLEX = icons/stax_app_$(COIN).gif
```

Comment:
Should be changed before merge?

Reply:
I still dont have icons, so it will be kept unfortunately.
But it deserves a comment for sure !